BNB Smart Chain/bsc-starter/.github/workflows/pr.yml

```
@@ -18,3 +18,7 @@ jobs:
         run: yarn codegen
       - name: Build
         run: yarn build
+      - name: Install subql-node
+        run: yarn global add @subql/node
```

Yarn global is not recommended 
```suggestion
        run: npm i -g @subql/node-ethereum
```